src/custom/theme/cowSwapTheme.tsx

```diff
@@ -16,6 +14,13 @@ import {
   ThemedGlobalStyle as ThemedGlobalStyleUniswap
 } from '@src/theme'
 import { useIsDarkMode } from 'state/user/hooks'
+import { cowSwapBackground, cowSwapLogo } from './cowSwapAssets'
+
+// Modal override items
+import { HeaderText } from '@src/components/WalletModal/Option'
```

Comment:
this path doesn't need @src or does it?

Reply:
My understanding is, that `@src` pulls the original component, where we export `HeaderText`. That's the one I want to target and override. We don't have a `custom` component with a HeaderText.

Reply:
yup then fine